tests/testing/LocalBearTestHelperTest.py

```diff
@@ -0,0 +1,43 @@
+from queue import Queue
+import unittest
+
+from tests.test_bears.TestBear import (FirstTestBear,
+                                       SecondTestBear,
+                                       ThirdTestBear)
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.testing.LocalBearTestHelper import verify_local_bear, execute_bear
+
+
+files = (("Everything is invalid/valid/raises error",),)
```

Comment:
but the above tests, wont work inside the setUp function! , `verify_local_bear` won't work